org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/exporting/SimpleNamesFragment2.java

```diff
@@ -14,12 +14,17 @@
 import org.eclipse.xtext.xtext.generator.model.GuiceModuleAccess;
 import org.eclipse.xtext.xtext.generator.model.TypeReference;
 
-@SuppressWarnings("all")
 public class SimpleNamesFragment2 extends AbstractXtextGeneratorFragment {
-  @Override
-  public void generate() {
-    new GuiceModuleAccess.BindingFactory().addfinalTypeToType(TypeReference.typeRef(IQualifiedNameProvider.class), TypeReference.typeRef(SimpleNameProvider.class)).contributeTo(this.getLanguage().getRuntimeGenModule());
-    new GuiceModuleAccess.BindingFactory().addTypeToType(TypeReference.typeRef("org.eclipse.xtext.ui.refactoring.IDependentElementsCalculator"), 
-      TypeReference.typeRef("org.eclipse.xtext.ui.refactoring.impl.DefaultDependentElementsCalculator")).contributeTo(this.getLanguage().getEclipsePluginGenModule());
-  }
+	@Override
+	public void generate() {
+		new GuiceModuleAccess.BindingFactory()
+				.addfinalTypeToType(TypeReference.typeRef(IQualifiedNameProvider.class),
```

Comment:
meh ... `addfinal..` should be `addFinal..`

Reply:
no idea why ....

Reply:
The feedback was unrelated.